hooks/useCreateProposal.ts

```diff
@@ -44,7 +45,8 @@ export default function useCreateProposal() {
 
     const defaultProposalMint =
       !mint?.supply.isZero() ||
-      realm?.account.config.useMaxCommunityVoterWeightAddin
+      realm?.account.config.useMaxCommunityVoterWeightAddin ||
+      true
```

Comment:
why true in here ?

Reply:
I think `useMaxCommunityVoterWeightAddin` is a mistake, so I have corrected it to `useCommunityVoterWeightAddin`. Thoughts?

Reply:
yes you are right version with useCommunityvoterWeightAddin will be better in most plugin cases please replace it in this files too.

<img width="680" alt="Zrzut ekranu 2022-06-11 o 21 38 59" src="https://user-images.githubusercontent.com/15833026/173202481-9186e688-eb1c-49b8-a821-c1c618778219.png">